libraries/testAutomation/testAutomation/test_runner.py

```diff
@@ -181,6 +181,16 @@ def cleanup_clusters(self) -> None:
             if pattern.match(configmap) and configmap.endswith("migration-config"):
                 self.k8s_service.delete_configmap(configmap_name=configmap)
 
+        # Cleanup non-Helm Kubernetes resources (ES 1.x and 2.x)
+        try:
+            self.k8s_service.exec_migration_console_cmd([
+                "kubectl", "delete", "all,configmap,secret",
+                "-l", "migration-test=true",
+                "--ignore-not-found"
+            ])
```

Comment:
We shouldn't need to run these commands on the migration console, check K8s service `run_command` function and look at is usages. Could also create a new function there that wraps this logic.